java/micronaut/gradle.properties

```diff
@@ -0,0 +1 @@
+micronautVersion=1.2.5
```

Comment:
could it be a range ?
using `1.2.5` is too strict, I do not know `gradle`, but I think using
```ini
micronautVersion=1.2.+
```
should work as expected

Reply:
sure i'll update it